### ISSUE TITLE
Please allow CORS on your API

### ISSUE THREAD

**Issue by SdtBarbarossa** [jedi-gg/swgoh.gg.public#275]
Your API is not allowing CORS. These makes it hard to call it via Javascript. I need to go over cors-anywere proxy. Could you be so kind to allow CORS?

**Comment by danpaulson** [comment]
Could you expand on what this proxy is aiming to accomplish?

**Comment by SdtBarbarossa** [comment]
Well for that to explain let me explain CORS:

CORS blocks every AJAX call (client-side-calls) that isnt called from same origin. That means if i call your API from e.g. localhost/ and your API is on swgoh.gg then the call will be blocked. if i would call from swgoh.gg/oons to your swgoh.gg/api/characters then the call would go through because swgoh.gg == swgoh.gg so "same origin". 
localhost/ != swgoh.gg so "not same origin" wich will be blocked.

This CORS policy doesnt apply to server side calls. Meaning if i call it from my localhost/ SERVERSIDE the api will be reachable. If i call it from clientside code the api will block.

There are 2 ways to allow API calls from client side:

1. JSONP instead of JSON. This is because JSONP is allowed through CORS policy 

2. Allow CORS on the api. I can not tell you how this can be done on django.
On IIS you do it in webconfig with:

 <system.webServer>
   <httpProtocol>
     <customHeaders>
       <add name="Access-Control-Allow-Origin" value="*" />
     </customHeaders>
   </httpProtocol>
 </system.webServer>

The * means that API can be called from everywhere via clientside-code.

So what does the proxy? Its simple.... i use the clientside code to call the proxy ( wich allows cors ) and then the proxy server will do a serverside call to your API ( wich is not affected by CORS ) and then return the result to my clientside code.

So basically:
ClientsideAJAX->Proxy->yourAPI->Proxy->ClientsideAJAX

Faster would be:
ClientsideAJAX->yourAPI->ClientsideAJAX

so if you would allow CORS just on the APIs you want to share that would be awesome!

Thank you for reading that much =)
